data-raw/babynames_sample.csv

```diff
@@ -0,0 +1,55 @@
+year,sex,name,n,prop
+1898,F,Catherine,1705,0.00621936
+1901,F,Ruth,3974,0.01563158
+1910,F,Mattie,1687,0.0040212
+1914,M,Stanley,3559,0.00520836
+1922,M,Gene,1089,9.6778e-4
+1923,F,Billie,1247,9.9564e-4
+1926,F,Louise,7983,0.00648944
+1928,F,Imogene,1315,0.00109999
+1929,F,Dorothy,31478,0.02719432
+1930,F,Ada,1040,8.9161e-4
+1932,F,Constance,2492,0.00225271
+1950,F,Katherine,6204,0.00352777
+1953,M,Lonnie,1923,9.609e-4
+1954,M,Darryl,1330,6.4301e-4
+1955,M,Jon,3054,0.00146153
+1959,M,Don,3620,0.00167109
+1960,M,Jackie,1367,6.3121e-4
+1961,M,Tom,4720,0.00218928
+1963,F,Allison,1434,7.2135e-4
+1964,M,Calvin,2253,0.00111131
+1965,M,Brian,24555,0.01295635
+1968,F,Andrea,7089,0.00414671
+1969,F,Annette,3105,0.00176148
+1970,M,Steven,26821,0.01407467
+1973,M,Billy,3444,0.00213355
+1974,F,Shawn,1271,8.1155e-4
+1976,F,Rebecca,14491,0.00921899
+1976,M,Christian,2267,0.00138805
+1979,F,Ruth,1201,6.9699e-4
+1980,F,Rhonda,1534,8.6162e-4
+1980,M,Adam,18873,0.01017399
+1984,F,Beth,1632,9.0521e-4
+1985,M,Isaac,1670,8.6806e-4
+1989,F,Meredith,1700,8.5346e-4
+1990,M,Drew,1953,9.0788e-4
+1991,M,Preston,1426,6.729e-4
+1993,F,Valerie,2176,0.00110391
+1994,F,Chelsey,1654,8.4863e-4
+1995,F,Kassandra,1607,8.3645e-4
+1996,F,Kaitlin,3135,0.00163547
+1996,M,Allen,1467,7.3227e-4
+1998,M,Peter,3386,0.00167034
+2000,F,Haylee,1261,6.3212e-4
+2002,F,Adrianna,1445,7.3203e-4
+2002,M,Francisco,2741,0.00132702
+2003,M,Malik,1505,7.1665e-4
+2005,M,Raul,1332,6.2652e-4
+2007,M,Griffin,1485,6.7091e-4
+2010,F,Andrea,3567,0.00182149
+2010,M,Kyler,1284,6.2569e-4
+2011,F,Addison,9297,0.0048045300000000004
```

Comment:
Zap these too?

Reply:
That is the output after zapsmall() with default options.  c.f. https://github.com/hadley/babynames/pull/30/commits/a4a05dbacab63972491b4faaacf1ff5cbdbcd3fb

Reply:
It looks like the logic in zapsmall is causing it to round to 8 places rather than 7 (log_10 of a value less than 1 is negative, in this case log_10(.08) ~= -1.08).  Perhaps this interacts with some logic inside of write_csv?  Why is this important?

Reply:
It's probably not important; it's just my sense of aesthetics. Don't worry about it.